install/ufw.sh

```diff
@@ -0,0 +1 @@
+sudo ufw enable
```

Comment:
May be nice to `sudo ufw status verbose` so people can see the firewall status on install.